include/aws/checksums/private/crc_util.h

```diff
@@ -31,7 +31,9 @@ static inline uint32_t aws_bswap32_if_be(uint32_t x) {
 
 #if _MSC_VER
     return _byteswap_ulong(x);
-#elif defined(__GNUC__) || defined(__clang__)
+#elif defined(__GNUC__) && (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 8))
```

Comment:
Do we need this level of fine-grained control? Should we just say GNUC >= 5? Also, we should add a comment.

Reply:
just picking the lowest version we test with. its available starting with 4.2. 
do you feel strongly about commenting? i initially wrote one, but decided to remove it, cause it seems obvious that its just checking for a version that supports a given intrinsic

Reply:
No; we can mention that it's supported starting with 4.2, and 4.8 is the lowest version we test with. Up to you.